app/commands/source/version_changelog_command.rb

```diff
@@ -0,0 +1,141 @@
+# frozen_string_literal: true
+
+module Source
+  class VersionChangelogCommand < ApplicationCommand
+    usage do
+      no_command
+    end
+
+    option :version do
+      long "--version string"
+      desc "The version of the full_names.yml file to compare"
+    end
+
+    def execute
+      version = params[:version] || latest_version
+      frame("Processing categories for version #{version}") do
+        ensure_exports_directory
+        delete_previous_exports(version)
+        process_category_tree
+        compare_and_export(version)
+      end
+    end
+
+    private
+
+    def latest_version
+      # Get the list of directories and sort them to find the latest version
+      versions = Dir.glob('data/integrations/shopify/*').select do |f|
+        File.directory?(f) && f.match(/\d{4}-\d{2}/)
+      end.map do |f|
+        File.basename(f)
+      end.sort
+
+      versions.last
+    end
+
+    def ensure_exports_directory
+      unless Dir.exist?('exports')
```

Comment:
We probably shouldn't commit this directory right? If not we should add `/exports` to the gitlog.